### PR TITLE
Rd multiple browser support

### PR DESCRIPTION
**Pull request by rmdaggett**
WIP for https://github.com/DramaFever/testing/issues/546 to allow parallelized cross-browser test runs on saucelabs. you will need to have browser capabilities set in your `sauce_config.py` module in the tests directory. 

```
BROWSERS = [{
    'platform': 'Windows 10',
    'browserName': 'chrome',
    'version': '58.0',
    'screenResolution': '1920x1080'
}, {
    'platform': 'Windows 10',
    'browserName': 'internet explorer',
    'version': '11.103',
    'screenResolution': '1920x1080'
}, {
    'platform': 'Windows 10',
    'browserName': 'firefox',
    'version': '42.0',
    'screenResolution': '1920x1080'
}]
```

after you've added some capabilities to your sauce config, run some tests via `sst-remote -d path/to/tests/ -u saucelabs -c <concurrency_num> -t per_case` to see everything in action

@DramaFever/qa

### CODE REVIEW

File: src/sst/testrail_helper.py [hunk 1]
```diff
@@ -15,20 +14,22 @@ def __init__(self, url, user, password, project_id):
...
         self.project_id = project_id
         self.run_results = []
-        self.run_id = None
+        self.run_ids = []
 
-    def create_test_run(self, case_ids):
-        tz = timezone('US/Eastern')
-        time = datetime.now(tz).time().strftime("%I:%M %p")
+    def create_test_run(self, case_ids, platform):
+        time = datetime.now().time().strftime("%I:%M %p")
```
Comment (rmdaggett) [inline src/sst/testrail_helper.py]: Add timezone back in?

**Comment by Bassoon08** [comment]
I mentioned this in person a few days ago, but noting here: it might be nice to create a TestRail Test Plan, and group all the Runs together in there, instead of having the Runs separated. Here's an example of a manual Test Plan which contains Test Runs of separate browsers, which represents testing for a single release: https://dramafever.testrail.net/index.php?/runs/view/2958

Here's a link to the TestRail API docs with details on the `add_plan` method where you can include a list of runs: http://docs.gurock.com/testrail-api2/reference-plans

**Comment by rmdaggett** [comment]
@Bassoon08 -- Good call! I'm pushing some commits up now. 

image

**Comment by rmdaggett** [comment]
Just as an update, this is still a WIP. I need to fix the way results are posted for local test runs .

**Comment by Bassoon08** [comment]
Hey Ryan, this looks good for running tests remotely and locally as well. Not sure if this is still a WIP, but sending the TestRail results `per_suite` is not working. I don't know if we still need that and could just remove that param/functionality and just have the `-t` flag always send `per_case`  - what do you think?

**Comment by rmdaggett** [comment]
Yeah, I think having the option in the beginning was nice but ultimately unnecessary and compounds the cost of making changes like this. I'm in favor of just removing `per_suite` and having the default functionality be `per_case`.

**Comment by Bassoon08** [comment]
@rmdaggett  Yes, agreed!

**Comment by rmdaggett** [comment]
@Bassoon08 done!

**Comment by Bassoon08** [comment]
Thanks Ryan this LGTM!

**Pull request merged**